### PR TITLE
ENH: Changed the takepeds to automatically pick a station

### PR DESCRIPTION
**Pull request by aoluwade**
## Description

Used  `get_info --getstation` to get the current station.
Added a usage function.
## Motivation and Context

It automatically gets the station for the user as opposed to the default station 0 before.
## How Has This Been Tested?

Tested on cxi-daq and cxi-monitor and it works fine.
## Where Has This Been Documented?

Added a usage fucntion in the script

### CODE REVIEW

**Review by silkenelson — APPROVED** [review]
While the commit message had be worried the '-r' was gone, the code looks good to me. I assume this was tested in the two CXI endstations?

**Review by ZryletTC — CHANGES_REQUESTED** [review]
Waiting on tests...
Could you also please add a usage? (look at other scripts for examples)

**Comment by aoluwade** [comment]
> 
A usage function has been added.

**Comment by aoluwade** [comment]
> Waiting on tests... Could you also please add a usage? (look at other scripts for examples)

Tested on cxi-daq and cxi-monitor and it works.

File: scripts/takepeds [hunk 1]
```diff
@@ -1,9 +1,25 @@
 #!/bin/bash
 
+usage()
+{
+cat << EOF
+usage: $0
+
+Generates the background (dark) image and the noise values for each pixel, and posts to the elog.
```
Comment (ZryletTC) [inline scripts/takepeds:8]:
```suggestion
Takes a run with dark images for use in pedestals, and posts to the elog.
```

**Review by ZryletTC — APPROVED** [review]

**Pull request merged**